snap/README.md

```diff
@@ -84,19 +131,20 @@ All services which are installed on the system as systemd units, which if enable
 
 All default configuration files are shipped with the snap inside `$SNAP/config`, however because `$SNAP` isn't writable, all of the config files are copied during snap installation (specifically during the install hook, see `snap/hooks/install` in this repository) to `$SNAP_DATA/config`.
 
-Note - as the core-config-seed was removed as part of the Geneva release, services self-seed their configuration on startup. This means that if a service is
-started by default in the snap, the only way to change configuration is to use the Consul UI or [kv REST API](https://www.consul.io/api/kv.html). Services that
-aren't started by default (see above) *will* pickup any changes made to their config files when enabled.
+**Note** - `$SNAP` resolves to the path `/snap/edgexfoundry/current/` and `$SNAP_DATA` resolves to `/var/snap/edgexfoundry/current`.
 
-```bash
-$ sudo snap restart edgexfoundry
-```
-
-### Viewing logs
+In the Geneva release of EdgeX, services were changed such that each became responsible for "self-seeding" its own configuration to Consul.
+Currently the only way to effect configuration changes for services that are auto-started (e.g. Core Data, Core Metadata) is to change
+configuration directly via Consul's UI or [kv REST API](https://www.consul.io/api/kv.html). Changes made to configuration in Consul
+require services to be restarted in order for the changes to take effect; the one exception are changes made to configuration items in
+a service's ```[Writable]``` section. Services that aren't started by default (see above) *will* pickup any changes made to their config
+files when started.
 
```

Comment:
I think the above block can be removed:

> In the Geneva release of EdgeX, services were changed such that each became responsible for "self-seeding" its own configuration to Consul.
> Currently the only way to effect configuration changes for services that are auto-started (e.g. Core Data, Core Metadata) is to change
> 

This doesn't really apply anymore as we now have rich config hook support...

Reply:
Rewrote that section to link it into the following configuration overrides section.